File: Administrativo/spring-erp-servicio/net/royal/spring/contabilidad/dominio/filtro/FiltroPaginacionAcCostcentermst.cs
﻿using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.contabilidad.dominio.filtro
{
    public class FiltroPaginacionAcCostcentermst
    {
        public String Codigo { get; set; }
        public String Nombre { get; set; }
        public String Estado { get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroPaginacionAcCostcentermst()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/selectores/centroCosto/dominio/filtropaginacionaccostcentermst.ts
import { ParametroPaginacionGenerico } from '../../../dominio/ParametroPaginacionGenerico';

export class FiltroPaginacionAcCostcentermst {
    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;
    codigo: string;
    nombre: string;
    estado: string;
}
